### DISCUSSION TITLE
Spaces in path - help needed

### DISCUSSION THREAD

**Discussion by MarGeb1** [microsoft/winget-cli#1296] [Q&A] [answered]
Hi there,

this is my working command line for installing Firefox:

winget install Mozilla.Firefox --override "Install_Directory_Path=F:\Firefox\ /passive" 

But when I change the install path to something with spaces:

winget install Mozilla.Firefox --override "Install_Directory_Path=F:\Mozilla Firefox\ /passive"

it doesn't work anymore and I face the MSI help.

I tried to include the path in single or double quotation marks but it didn't help.

So who can help me?

Markus

BTW: It seems that the interactive-switch does not work with Firefox?

**Comment by Masamune3210** [comment]
Try quoting the path, as that is usually required when interacting with the terminal or passing args when dealing with spaces

**Reply by MarGeb1** [reply]
As I wrote before, I tried to include the path in single or double quotation marks but it didn't work.

When I do a local cmd install with the Mozilla.Firefox.90.0.msi downloaded by winget

`Mozilla.Firefox.90.0.msi Install_Directory_Path="D:\Mozilla Firefox\" /passive` 

works fine.

Due to the fact, that the problem is not limited to Firefox only, maybe there's a problem with the --override command?

Markus

**Reply by denelon** [reply]
@MarGeb1 I've seen a couple of instances with paths (namely those with spaces) that they should be the last argument, and don't need to be quoted/escaped in some cases. I don't know if this holds true in this case or not. I also tried to `winget install firefox --location <path>` but that didn't seem to get the desired results either.

**Reply by MarGeb1** [reply]
Hi denelon,

I tried to change the arguments and used single, double or no quotation marks again but without success. Maybe it should be reported as a bug?

Markus

**Reply by denelon** [reply]
@MarGeb1 go ahead and report it as a bug. I'll see if I can get someone to look at it to determine if there is something we're missing or if there is a bug we need to fix 👍

**Reply by ImJoakim** [reply]
> @Margeb go ahead and report it as a bug. I'll see if I can get someone to look at it to determine if there is something we're missing or if there is a bug we need to fix 👍

@denelon I believe you tagged the wrong person, forgot that number 1 at the end (@MarGeb1).

**Comment by MarGeb1** [comment, marked as answer]
Thanx to @yao-msft the solution can be found [here](https://github.com/microsoft/winget-cli/issues/1317#issuecomment-885962937)
